tests/test_s3.py

```diff
@@ -1059,3 +1059,30 @@ def test_s3_remove_tag(self):
         tags = client.get_bucket_tagging(Bucket=bname)
         tag_map = {t['Key']: t['Value'] for t in tags.get('TagSet', {})}
         self.assertTrue('maid_status' not in tag_map)
+
+    def test_hosts_website(self):
+        self.patch(s3, 'S3_AUGMENT_TABLE', [
+            ('get_bucket_website', 'Website', None, None)])
+        session_factory = self.replay_flight_data('test_s3_hosts_website')
+        session = session_factory()
+        client = session.client('s3')
+        bname = 'custodian-static-website-test'
```

Comment:
there's something missing here, you've got recorded data for enabling the website hosting on the bucket (PutBucketWebsite), but i don't see a corresponding api call here when creating the bucket to enable the website hosting. that said the recorded data does get to the correct coverage, but it would be nice to have that api call here so that it can be re-recorded.